pages/index.tsx

```diff
@@ -144,6 +144,34 @@ export default function Home({ topAlgorithms, featuredAlgorithms }) {
                     please fork the repository, work on your changes and then
                     submit them as a pull request.
                   </Typography>
+                  <Typography variant="h5" className={classes.title}>
+                    Donate
+                  </Typography>
+                  <Typography>
+                    Another way you can support us is to make a donation via
+                    Liberapay. Even a small donation is much appreciated.
```

Comment:
You can maybe here add a link to [Liberapay](https://liberapay.com) official website.

Reply:
47dc26c6dc8fa75eb26479fc22b7438b1c602935